_includes/glossary/README.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Glossary
+
+To write a glossary snippet, start with a copy of the template in this directory, `_glossary-template.md`. Make sure to provide (or omit) values for the following fields:
+
+* (Required) `term`
+* (Optional) `formerly` and `related`. If you do not provide these values, remove the fields.
```

Comment:
should `formerly` and `related` be their own separate lines?

Also (depending who merges first) we'll just need to make sure that there is an additional (Required) bullet point for `tags`?